File: exercism/go/pangram/pangram.go
package pangram

func IsPangram(input string) bool {
	var lettersBitset int
	for i := range input {
		chr := input[i]
		if chr >= 'a' && chr <= 'z' {
			lettersBitset |= 1 << (chr - 'a')
		} else if chr >= 'A' && chr <= 'Z' {
			lettersBitset |= 1 << (chr - 'A')
		}
	}
	return lettersBitset == 0x3ffffff
}

File: exercism/kotlin/pangram/src/main/kotlin/Pangram.kt
import java.util.BitSet

object Pangram {

    private const val alphabetSize = 26

    fun isPangram(input: String): Boolean {
        val letters = BitSet(alphabetSize)
        input
            .map(Char::toLowerCase)
            .filter(Char::isLetter)
            .forEach { letters.set(it.toInt()) }
        return letters.cardinality() == alphabetSize
    }
}
